Library/Formula/cloc.rb

```diff
@@ -11,6 +11,12 @@ class Cloc < Formula
     sha256 "a4c159ceb7f5f67d7925864a8e4f31e2c43fe2dee356afdf931ebc1000ca11a3" => :mountain_lion
   end
 
+  # remove deprecated defined(@array) to support perl 5.22 (bug 135)
+  # remove "no warnings 'deprecated';" -- no longer relevant
+  # don't write language definition for "(unknown)"
+  # http://sourceforge.net/p/cloc/bugs/135/
+  patch :DATA
```

Comment:
Can’t you use the linked patch? https://bugs.archlinux.org/task/45494?getfile=13174